generator/uscore/uscore_generator.rb

```diff
@@ -309,7 +329,10 @@ def create_resource_profile_test(sequence)
         test = {
           tests_that: "#{sequence[:resource]} resources associated with Patient conform to US Core R4 profiles",
           index: sequence[:tests].length + 1,
-          link: sequence[:profile]
+          link: sequence[:profile],
+          description: %(
+            This test checks if the resources returned from prior searches conform to the US Core profiles.
+            This includes checking for missing data elements and valueset verification.)
```

Comment:
The closing parenthesis should be on the following line for consistency.

Reply:
Fixed